acceptance/suites/tests/010-puppetserver-cli/subcommand/SERVER-297_common_behavior.rb

```diff
@@ -13,6 +15,18 @@
   assert(env['JARS_NO_REQUIRE'], "JARS_NO_REQUIRE missing")
 end
 
+step "ruby: Check that proxy env-variables are present"
+on(master, "#{proxy_env_vars} puppetserver ruby -rjson -e 'puts JSON.pretty_generate(ENV.to_hash)'") do
+  env = JSON.parse(stdout)
+  assert(env['HTTP_PROXY'], "HTTP_PROXY is missing")
+  assert(env['http_proxy'], "http_proxy is missing")
+  assert(env['HTTPS_PROXY'], "HTTPS_PROXY is missing")
+  assert(env['https_proxy'], "https_proxy is missing")
+  assert(env['NO_PROXY'], "NO_PROXY is missing")
+  assert(env['no_proxy'], "no_proxy is missing")
+  assert_nil(env['RANDOM_VARIABLE'], "RANDOM_VARIABLE is present and should not be")
```

Comment:
I think my plan had been to add that variable as an environment variable in the list and then assert that it wasn't present, since it doesn't seem like we have anything that tests that we're not just allowing all variables through. But I didn't include half of the behavior necessary to make such a test useful. Also I don't think that that should necessarily be part of this test?